src/components/Main_Destination/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import pagesData from '../../../assets/pagesData';
+
+const Main_Destination = () => {
```

Comment:
JS uses the camel case naming convention. You should not use underscores.

```suggestion
const MainDestination = () => {
```